src/assets/scripts/service-k3s.initd

```diff
@@ -25,6 +25,9 @@ ENGINE="${ENGINE:-containerd}"
 depend() {
   after network-online
   want cgroups
+  if [ "${ENGINE}" == "moby" ]; then
```

Comment:
You don't need the conditional, you can just add `after cri-dockerd`.  If `cri-dockerd` is not configured for this runlevel, then the directive is simply ignored.

`after` only affects ordering. You use `need` to specify a dependency.

Reply:
https://github.com/OpenRC/openrc/blob/master/service-script-guide.md#the-depend-function

Reply:
Well, and we'd also need to manually start `cri-dockerd` somewhere (in `lima.ts` & `wsl.ts`), much like we currently do for `docker`.  Alternatively, add `need cri-dockerd` here (in which case we'll need to run `rc-update --update` before we start `k3s`; we used to have have, not sure if we took it out already).

(In more detail: OpenRC has a cache of the dependencies, so if we change them based on configs like `ENGINE`, we'll need to run `rc-update --update` to flush the cache.)